comprehensive-review/solutions/garden-back/src/main/java/com/redhat/garden/back/resource/SensorResource.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.garden.back.resource;
+
+import com.redhat.garden.back.measurement.SensorMeasurement;
+import com.redhat.garden.back.measurement.SensorMeasurementEnriched;
+import io.smallrye.reactive.messaging.annotations.Broadcast;
+import org.eclipse.microprofile.reactive.messaging.Acknowledgment;
+import org.eclipse.microprofile.reactive.messaging.Channel;
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+import org.jboss.resteasy.annotations.SseElementType;
+import org.reactivestreams.Publisher;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("sensor")
+public class SensorResource {
+
+    @Inject @Channel("in-memory-garden-enriched-sensor-measurements")
+    Publisher<SensorMeasurementEnriched> enrichedSensorMeasurements;
+
+    @Inject @Channel("in-memory-garden-raw-sensor-measurements")
+    Publisher<SensorMeasurement> rawSensorMeasurements;
+
+    // Event processors ------------------------------------------------------------------------------------------------
+
+    @Incoming("garden-sensor-measurements-enriched")
+    @Outgoing("in-memory-garden-enriched-sensor-measurements")
```

Comment:
Maybe we can use shorter channel names. Since we are setting the topic name in the configuration we can use shorter names I guess

Reply:
Same for the other channel names